OracleEssbase/dockerfiles/21.1.0/Dockerfile

```diff
@@ -0,0 +1,125 @@
+#
+# Copyright (c) 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+#
+# Base image
+#
+ARG FMW_INFRA_IMAGE=oracle/fmw-infrastructure:12.2.1.4
+FROM ${FMW_INFRA_IMAGE} as base
+
+# Labels
+# ------
+LABEL "provider"="Oracle"                                             \
+      "issues"="https://github.com/oracle/docker-images/issues"       \
+      "maintainer"="Anil Arora <anil.arora@oracle.com>"               \
+      "volume.data"="/u01/data"                                       \
+      "volume.config"="/u01/config"                                   \
+      "volume.temp"="/u01/tmp"                                        \
+      "port.node.manager"="9556"                                      \
+      "port.admin.server"="7001"                                      \
+      "port.admin.server.ssl"="7002"                                  \
+      "port.managed.server"="9000"                                    \
+      "port.managed.server.ssl"="9001"                                \
+      "port.agent"="1423"                                             \
+      "port.agent.ssl"="6423"                                         \
+      "port.eas.server"="9100"                                        \
+      "port.eas.server.ssl"="9101"
+
+USER root
+RUN mkdir /u01/config /u01/data /u01/tmp && \
+    chmod 750 /u01/config /u01/data /u01/tmp && \
+    chown oracle:oracle /u01/config /u01/data /u01/tmp && \
+    rm -rf /u01/container-scripts/* && \
+    rm -rf /u01/oracle/container-scripts
+
+# Essbase Prerequisites
+RUN yum install -y gcc gcc-c++ \
+                   libgcc libgcc.i686 \
+                   numactl \
+                   hostname && \
+    yum clean all && \
```

Comment:
You don't need this command because you're running `rm -rf /var/cache/yum` which does the same thing.

Reply:
Removed yum clean all from the command.